tests.py

```diff
@@ -238,5 +287,15 @@ def test_suite():
         doctest.DocTestSuite(),
     ])
 
+
+# Test suite rules.
+
+
+def suite():
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.TestLoader().loadTestsFromTestCase(ShowGraphTest))
+    suite.addTest(doc_test_suite())
+    return suite
+
 if __name__ == '__main__':
-    unittest.main(defaultTest='test_suite')
+    unittest.main(defaultTest='suite')
```

Comment:
There's no need for two functions returning test suites.  Modify the existing test_suite() to do something like

```python
doctests = find_doctests()
return unittest.TestSuite([
         unittest.defaultTestLoader.loadTestsFromName(__name__),
         doctest.DocFileSuite(setUp=setUp, tearDown=tearDown,
                              optionflags=doctest.ELLIPSIS,
                              checker=IgnoreNodeCountChecker(),
                              *doctests),
         doctest.DocTestSuite(),
     ])
```